openbb_platform/core/openbb_core/app/static/package_builder.py

```diff
@@ -915,19 +927,14 @@ def get_field_type(
                 .replace("pydantic.types.", "")
                 .replace("datetime.datetime", "datetime")
                 .replace("datetime.date", "date")
+                .replace("datetime.time", "time")
```

Comment:
For future reference: we need to be careful with these things, since it's not a Python type, the CLI might break trying to parse unexpected types.

Reply:
I think that it should have the complete type i.e. as `datetime.<type>`. Thoughts @montezdesousa ?